sdk/mixedreality/mixedreality-authentication/README.md

```diff
@@ -0,0 +1,206 @@
+
+```bash
+npm install @azure/identity
+```
+
+You will also need to [register a new AAD application][register_aad_app] and grant access to your Mixed Reality resource
+by assigning the appropriate role for your Mixed Reality service to your service principal.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+```js
+const { MixedRealityStsClient } = require("@azure/mixedreality-authentication");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+const client = new MixedRealityStsClient(accountId, accountDomain, new DefaultAzureCredential());
+```
+
+## Key concepts
+
+### MixedRealityStsClient
+
+The `MixedRealityStsClient` is the client library used to access the Mixed Reality STS to get an access token.
+
+Tokens obtained from the Mixed Reality STS have a lifetime of **24 hours**.
+
+### Return Value
+
+The return value for a successful call to `getToken` is an `GetTokenResponse`, which is an `AccessToken` from
+[@azure/core-http](https://www.npmjs.com/package/@azure/core-http) that also contains HTTP request and response
+details in the `_response` field.
+
+## Examples
+
+### Retrieve an access token
+
+```js
+import { AzureKeyCredential } from "@azure/core-auth";
+
+const { MixedRealityStsClient } = require("@azure/mixedreality-authentication");
+
+const accountId = "<ACCOUNTD ID>";
+const accountDomain = "<ACCOUNT_DOMAIN>";
+const accountKey = "<ACCOUNT_KEY>";
+
+const client = new MixedRealityStsClient(
+  accountId,
+  accountDomain,
+  new AzureKeyCredential(accountKey)
+);
+
+const token = await client.getToken();
+```
+
+See the authentication examples [above](#authenticate-the-client) or [Azure Identity][azure_identity] for more complex
+authentication scenarios.
+
+#### Using the access token in a Mixed Reality client library
+
+Some Mixed Reality client libraries might accept an access token in place of a credential. For example:
+
+```js
+// GetMixedRealityAccessTokenFromWebService is a hypothetical method that retrieves
+// a Mixed Reality access token from a web service. The web service would use the
+// MixedRealityStsClient and credentials to obtain an access token to be returned
+// to the client.
+const accessToken = await GetMixedRealityAccessTokenFromWebService();
+
+const account = new SpatialAnchorsAccount(accountId, accountDomain);
+const client = new SpatialAnchorsClient(account, accessToken);
+```
+
+Note: The `SpatialAnchorsClient` usage above is hypothetical and may not reflect the actual library. Consult the
+documentation for the client library you're using to determine if and how this might be supported.
+
+## Troubleshooting
+
+### Enable logs
+
+You can set the following environment variable to see debug logs when using this library.
+
+- Getting debug logs from the Azure MixedRealityStsClient client library
+
+```bash
+export DEBUG=azure*
+```
```

Comment:
I see this is what we currently have in our template project, but I think this is out of date. The section should encourage use of the `AZURE_LOG_LEVEL` environment variable to enable logging.

Here's the same section from the identity package:

>### Logging
>
>Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
>
>```javascript
>import { setLogLevel } from "@azure/logger";
>
>setLogLevel("info");
>```

Reply:
Ah! Great info. I'll snag that. Thanks!